compiler/src/iree/compiler/Codegen/SPIRV/test/tile_and_vectorize_to_cooperative_ops.mlir

```diff
@@ -102,9 +102,9 @@ func.func @matmul_256x1024x128_div_add() attributes {translation_info = #transla
 //       CHECK:       vector.transfer_write %[[ZERO]], {{.+}} : vector<16x16xf16>, memref<16x16xf16, strided<[128, 1], offset: ?>>
 //       CHECK:   scf.for %{{.+}} = %[[C0]] to %[[C1024]] step %[[C32]]
 //       CHECK:     gpu.barrier
-//       CHECK:     vector.transfer_read {{.+}} vector<1x8xf16>
+//       CHECK:     vector.transfer_read {{.+}} vector<8xf16>
 //       CHECK:     vector.transfer_write
-//       CHECK:     vector.transfer_read {{.+}} vector<1x8xf16>
+//       CHECK:     vector.transfer_read {{.+}} vector<8xf16>
```

Comment:
test changes caused by switching pass to optimize-vector-transfer, which also drops vector unit dims.